daprdocs/content/en/operations/components/setup-bindings/supported-bindings/alicloudoss.md

```diff
@@ -65,18 +67,33 @@ By default, a random UUID is auto-generated as the object key. See below for Met
 
 **Saving to a random generated UUID file**
 
+{{< tabs Linux >}}
+
+{{% codetab %}}
+
 ```bash
 curl -d "{ \"operation\": \"create\", \"data\": \"Hello World\" }" http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
 ```
 
+{{% /codetab %}}
+
+{{< /tabs >}}
+
 <br />
 
 **Saving to a specific file**
+{{< tabs Linux >}}
+
+{{% codetab %}}
 
 ```bash
 curl -d "{ \"operation\": \"create\", \"data\": \"Hello World\", \"metadata\": { \"key\": \"my-key\" } }" http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
```

Comment:
Since this is escaped wouldn't this be the Windows Command Prompt command?

Reply:
@mukundansundar - You need to provide Windows and Linux tabs here. This example is for Windows. Add a Linux tab and remove the \ in front of the escaped " and this is the linux version. Can you add these in?